section4/exercises/burrito.rb

```diff
@@ -11,6 +11,17 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_topping
+    p "I want #{toppings}"
```

Comment:
For these methods, do some digging around ruby prototype methods to see what methods are baked into the ruby programming data that help you work with arrays. Right now, these methods print messages, but do not actually change attributes.